doc/devel/release_guide.rst

```diff
@@ -54,7 +54,7 @@ temporarily comment out the include and toctree glob; re-instate these after a
 release. Finally, make sure that the docs build cleanly ::
 
   pushd doc
-  python make.py html latex -n 16
+  make html latexpdf -n$(nproc)
```

Comment:
`O=-n$(nproc)`? Currently, this will build `html` and `latexpdf` in parallel, but I'm not sure Sphinx will like two runs occurring in parallel.

Reply:
Ooops, I didn't check this one.   Good eye: that doesn't run at all on my machine.

Glad someone more knowledgeable is reviewing!

Reply:
Good catch.

Reply:
I think you only fixed the one below that didn't have this comment. 😛

Reply:
try again...